src/setup/cortx_deploy

```diff
@@ -94,13 +106,17 @@ validate_config $CLUSTER_INFO
 
 export PATH=$PATH:/opt/seagate/provisioner/bin
 
+node_id=`cat /etc/machine-id`
+cluster_id=$(conf_get $(yaml_config $CLUSTER_INFO) "cluster>id")
+log $INFO "Starting deployment on node $node_id which is part of cluster $cluster_id."
+
 # Apply Cluster Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE";
+log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE' $FORCE_DEPLOY"
+cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE" $FORCE_DEPLOY;
```

Comment:
I think force option is -p $FORCE...

Reply:
I have added -o option in cortx_setup same as we have added for cortx_deploy.
```
parser.add_argument('-o', dest='override', action='store_true',
            help='config override')
```
cortx_deploy cmd:
```
/opt/seagate/cortx/provisioner/bin/cortx_deploy -f /root/auto-gen-cfgmap/ -c yaml:///etc/cortx/cluster.conf ;
or
/opt/seagate/cortx/provisioner/bin/cortx_deploy -f /root/auto-gen-cfgmap/ -c yaml:///etc/cortx/cluster.conf  -o;

```
cortx_setup cmd:
```
cortx_setup config apply -f yaml:///root/auto-gen-cfgmap//cluster.yaml -c 'yaml:///etc/cortx/cluster.conf'
or
cortx_setup config apply -f yaml:///root/auto-gen-cfgmap//cluster.yaml -c 'yaml:///etc/cortx/cluster.conf' -o
```